antora-ui-camel/src/css/doc.css

```diff
@@ -40,6 +40,16 @@
   .doc > h1.page:first-child {
     margin-top: 2.5rem;
   }
+
+  .release dl {
+    overflow: scroll;
+  }
```

Comment:
This needs to go to [antora-ui-camel/src/css/release.css](https://github.com/apache/camel-website/blob/master/antora-ui-camel/src/css/release.css)